sdk/core/azure-core/inc/azure/core/http/body_stream.hpp

```diff
@@ -134,7 +153,7 @@ namespace Azure { namespace Core { namespace Http {
 
     int64_t Length() const override { return this->m_length; }
 
-    int64_t Read(Context const& context, uint8_t* buffer, int64_t count) override;
+    int64_t OnRead(Context const& context, uint8_t* buffer, int64_t count) override;
```

Comment:
Why is OnRead public?  As a customer I expect to call `Read`

Reply:
Very good catch!
It should be protected, and I need to update this to all concrete BodyStream classes. THank you